game.project

```diff
@@ -1,6 +1,6 @@
 [project]
 title = Pokedex5E
-dependencies = https://github.com/britzl/monarch/archive/2.20.0.zip,https://github.com/Jerakin/gooey/archive/scale_with_layout_scrollbar.zip,https://github.com/defold/extension-safearea/archive/1.0.0.zip,https://github.com/GameAnalytics/GA-SDK-DEFOLD/archive/3.1.8.zip,https://github.com/selimanac/defold-random/archive/v1.2.1.zip,https://github.com/Jerakin/defsave/archive/master.zip,https://github.com/britzl/defold-clipboard/archive/1.1.zip,https://github.com/britzl/defold-lfs/archive/1.0.1.zip,https://github.com/Jerakin/editor-script-components/archive/master.zip,https://github.com/Jerakin/editor-script-atlas/archive/master.zip,https://github.com/Jerakin/editor-script-cleanup/archive/master.zip,https://github.com/AGulev/set_screen_position/archive/1.0.1.zip,https://github.com/Jerakin/PokemonTypes/archive/v1.0.2.zip,https://github.com/Jerakin/editor-script-check-dependencies-versions/archive/master.zip,https://github.com/GameAnalytics/defold-openssl/archive/1.0.0.zip,https://github.com/britzl/defold-sharing/archive/master.zip,https://github.com/andsve/def-diags/archive/master.zip
+dependencies = https://github.com/britzl/monarch/archive/2.20.0.zip,https://github.com/Jerakin/gooey/archive/scale_with_layout_scrollbar.zip,https://github.com/defold/extension-safearea/archive/1.0.0.zip,https://github.com/GameAnalytics/GA-SDK-DEFOLD/archive/3.1.8.zip,https://github.com/selimanac/defold-random/archive/v1.2.1.zip,https://github.com/Jerakin/defsave/archive/master.zip,https://github.com/britzl/defold-clipboard/archive/1.1.zip,https://github.com/britzl/defold-lfs/archive/1.0.1.zip,https://github.com/Jerakin/editor-script-components/archive/master.zip,https://github.com/Jerakin/editor-script-atlas/archive/master.zip,https://github.com/Jerakin/editor-script-cleanup/archive/master.zip,https://github.com/AGulev/set_screen_position/archive/1.0.1.zip,https://github.com/Jerakin/PokemonTypes/archive/v1.0.2.zip,https://github.com/Jerakin/editor-script-check-dependencies-versions/archive/master.zip,https://github.com/GameAnalytics/defold-openssl/archive/1.0.0.zip,https://github.com/britzl/defold-sharing/archive/master.zip,https://github.com/andsve/def-diags/archive/master.zip,https://github.com/britzl/defnet/archive/master.zip,https://github.com/britzl/defold-luasocket/archive/0.11.zip
```

Comment:
Should lock down defnet

Reply:
This is something that should be fixed I just don't know the right way of going about it.

A few notes:

- defnet hasn't seen a release in 2 years, so I am not confident that they will pull any PRs we send them
- the latest version has a major bug that prevents you from being able to send messages from the client to a server (see TCP server, here: https://github.com/britzl/defnet/compare/5.0.1...master?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8). This also makes me not confident the project is being properly maintained, as how can such a major bug exist without a release to fix it?
- I need to tweak p2p in order to send payload information along the broadcast so when finding local hosts you can get the group name and port. You can see the updated version of p2p in this review.
- I would rather a defnet fork be under your user than mine, because this is your project and I would prefer if it was all self-contained (I don't want to be the long-term owner for any of this)
- I do not know how to make a release, nor what a release version would look like if we did a release of the P5E version of defnet (version "p5e-1.0.0?")

Reply:
The owner of the repo is Britzl, he is the head of the Defold foundation. He have a major stake in that all libraries being connected to Defold is kept up to date and functioning. I am sure that if we create issues he will do them quickly (if relatively easy like creating a release) and good PRs will be merged within a week. I am of the same idea as you, I would rather not maintain it. :) That's why I would rather create a PR to DefNet and also as ask him to create a release. They are very helpful in general.